viewer/packages/camera-manager/src/CameraManagerHelper.ts

```diff
@@ -76,9 +103,16 @@ export class CameraManagerHelper {
    * Updates near and far plane of the camera based on the bounding box.
    * @param camera Used camera instance.
    * @param boundingBox Bounding box of all objects on the scene.
+   * @param nearAndFarPlaneBuffers to read from
+   * @param cameraFarBuffers
    */
-  static updateCameraNearAndFar(camera: PerspectiveCamera, boundingBox: Box3): void {
-    const { cameraPosition, cameraDirection, corners } = this._updateNearAndFarPlaneBuffers;
+  static updateCameraNearAndFar(
+    camera: PerspectiveCamera,
+    boundingBox: Box3,
+    nearAndFarPlaneBuffers: NearAndFarPlaneBuffers = CameraManagerHelper._updateNearAndFarPlaneBuffers,
+    cameraFarBuffers: CameraFarBuffers = CameraManagerHelper._calculateCameraFarBuffers
+  ): void {
+    const { cameraPosition, cameraDirection, corners } = nearAndFarPlaneBuffers;
```

Comment:
I think we can remove these extra parameters, and deprecate this function to suggest that users always use the instance-methods instead.